docker-compose.yml

```diff
@@ -11,20 +11,25 @@ app:
     - DATABASE_URL=postgres://tock_user@db/tock
     - RUNNING_IN_DOCKER=yup
     - DJANGO_SETTINGS_MODULE=tock.settings.dev
-  command: "python manage.py runserver 0.0.0.0:${EXTERNAL_PORT}"
+  command: "python manage.py runserver 0.0.0.0:1234"
```

Comment:
So this fixes a bug introduced in 267344707ed8275df1588d75c6084ad28eac60a0, which is my fault because I decided to make Django's runserver run on the same internal container port as the default `EXPOSED_PORT` that nginx is ultimately listening on.

To reduce the possibility of confusion, I've changed the Django runserver port to be 1234.